src/views/create-order/CreateOrderView.js

```diff
@@ -134,247 +137,302 @@ const CreateOrderView = (props: Props) => {
-              <br/>
-
+              <br />
 
               <Form.Label>Address Line 2</Form.Label>
               <Form.Control
-                  as={'input'}
-                  value={addressLine2}
-                  onChange={(event) => setAddressLine2(event.target.value)}
+                as={"input"}
+                value={addressLine2}
+                onChange={event => setAddressLine2(event.target.value)}
               />
-              <br/>
+              <br />
 
               <Form.Label>City</Form.Label>
               <Form.Control
-                  as={'input'}
-                  value={city}
-                  onChange={(event) => setCity(event.target.value)}
+                as={"input"}
+                value={city}
+                onChange={event => setCity(event.target.value)}
               />
-              <br/>
+              <br />
 
               <Form.Label>County</Form.Label>
               <Form.Control
-                  as={'input'}
-                  value={county}
-                  onChange={(event) => setCounty(event.target.value)}
+                as={"input"}
+                value={county}
+                onChange={event => setCounty(event.target.value)}
               />
-              <br/>
+              <br />
 
-              <Form.Label>Postcode <small>(required)</small></Form.Label>
+              <Form.Label>
+                Postcode <small>(required)</small>
+              </Form.Label>
               <Form.Control
-                  as={'input'}
-                  value={postCode}
-                  required
-                  onChange={(event) => setPostCode(event.target.value)}
+                as={"input"}
+                value={postCode}
+                required
+                onChange={event => setPostCode(event.target.value)}
               />
               <Form.Control.Feedback type="invalid">
                 We can't find your address without your postcode.
               </Form.Control.Feedback>
 
-              <br/>
+              <br />
 
-              <Form.Label>Phone Number <small>(required)</small></Form.Label>
+              <Form.Label>
+                Phone Number <small>(required)</small>
+              </Form.Label>
               <Form.Control
-                  as={'input'}
-                  value={phone}
-                  required
-                  onChange={(event) => setPhone(event.target.value)}
+                as={"input"}
+                value={phone}
+                required
+                onChange={event => setPhone(event.target.value)}
               />
               <Form.Control.Feedback type="invalid">
-                We need your phone number is required, what if we can't locate you?
+                We need your phone number is required, what if we can't locate
+                you?
               </Form.Control.Feedback>
-              <br/>
-
+              <br />
+              <Form.Label>Dietary Requirements</Form.Label>
```

Comment:
Is this only applicable to delivery only? Think we should add it to all orders maybe?

Reply:
If you look at line 246. Ive added it there aswell. Should it does apply to all orders

Reply:
Ahh I see, but I think we should add it under the "Iftar Packs" section, what you reckon?

Reply:
sure, that would work, would reduce duplication aswell. Ill make that change now